src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/MenuFlyoutTests/MenuFlyoutTests.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.MenuFlyoutTests
+{
+	[TestFixture]
+	class MenuFlyoutTests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void MenuFlyoutItem_ClickTest()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuFlyoutTests.MenuFlyoutItem_Click");
+
+			_app.WaitForElement(_app.Marked("mfiButton"));
+
+			TakeScreenshot("Initial");
+
+			// step 1: press button to show menu
+			_app.FastTap(_app.Marked("mfiButton"));
+
+			TakeScreenshot("menuShown");
+
+			// step 2: click MenuFlyoutItem
+			_app.FastTap(_app.Marked("mfiItem"));
+
+			// step 3: check result
+			_app.WaitForText(_app.Marked("mfiResult"), "success");
+
+			TakeScreenshot("AfterSuccess");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Simple_MenuFlyout()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuBarTests.SimpleMenuBar");
+
+			_app.WaitForElement(_app.Marked("fileMenu"));
+			var result = _app.Marked("result");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'result'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)